pyvo/dal/dbapi2.py

```diff
@@ -18,14 +12,14 @@
 __all__ = "STRING BINARY NUMBER DATETIME ROWID".split()
 
 
-class Error(StandardError):
+class Error(Exception):
```

Comment:
beyond the scope of this PR, but I'm not sure what use of these base classes have, why not use the standard python `Warning` and a sensible Exception class to subclass from?